README.md

```diff
@@ -26,9 +26,12 @@ That's it :)
 
 ### Create the video
 
-TODO (fix PYTHONPATH for other OS)
+TODO
 ```
-LANGUAGE=armenian PYTHONPATH=lib:objects python qarakusi.am.py 10674
+FOR LINUX, MACOS: LANGUAGE=armenian PYTHONPATH=lib:objects python qarakusi.am.py 10674
+
+FOR WINDOWS: $env:LANGUAGE='armenian'; $env:PYTHONPATH='lib;objects'; python qarakusi.am.py 10674
```

Comment:
Can you write the comments (e.g. For windows) outside the code block (outside 3 backquotes)?

Reply:
Thus, have 2 code blocks with different "description"s before them